src/index.html

```diff
@@ -22,6 +22,393 @@
+              href="#"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="catalog__container">
+        <article
+          data-qa="card"
+          class="catalog__cards catalog__cards--first"
+        >
+          <img
+            class="catalog__image"
+            src="images/imac.jpeg"
+            alt="image"
+          />
+          <div class="catalog__content">
+            <h2 class="catalog__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="catalog__code">Product code: 195434</p>
+            <div class="catalog__rate">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="catalog__review">Reviews: 5</p>
+            </div>
+            <div class="catalog__price-block">
+              <p class="catalog__price">Price:</p>
+              <h2 class="catalog__value">$2,199</h2>
+            </div>
+            <button
+              data-qa="card-hover"
+              class="catalog__button"
+              type="button"
+            >
+              BUY
+            </button>
+          </div>
+        </article>
+
+        <article
+          data-qa="card"
+          class="catalog__cards catalog__cards--second"
```

Comment:
Fix classes everywhere
```suggestion
          class="card"
```